Test fails due to string being quoted

Running the tests for this library produces one failure:

```
FAIL: schema_test.go:742: S.TestTimeDuration

schema_test.go:758:
    c.Assert(err.Error(), gc.Equals, "<path>: conversion to duration: time: invalid duration failure")
... obtained string = "<path>: conversion to duration: time: invalid duration \"failure\""
... expected string = "<path>: conversion to duration: time: invalid duration failure"

OOPS: 31 passed, 1 FAILED
--- FAIL: Test (0.01s)
```

It appears that the invalid string is now being quoted in the error message. (I'm not really a go developer, so I don't know if this is an expected change or not.)

The fix seems fairly straightforward:

```
diff --git a/schema_test.go b/schema_test.go
index 80ef302..667e9ed 100644
--- a/schema_test.go
+++ b/schema_test.go
@@ -755,7 +755,7 @@ func (s *S) TestTimeDuration(c *gc.C) {
        c.Assert(out, gc.Equals, value)
 
        out, err = sch.Coerce("failure", aPath)
-       c.Assert(err.Error(), gc.Equals, "<path>: conversion to duration: time: invalid duration failure")
+       c.Assert(err.Error(), gc.Equals, "<path>: conversion to duration: time: invalid duration \"failure\"")
 
        out, err = sch.Coerce(42, aPath)
        c.Assert(out, gc.IsNil)
```

Running on Debian sid, go version 1.17.2 linux/amd64.

--- a/schema_test.go
+++ b/schema_test.go
@@ -755,7 +755,7 @@ func (s *S) TestTimeDuration(c *gc.C) {
 	c.Assert(out, gc.Equals, value)
 
 	out, err = sch.Coerce("failure", aPath)
-	c.Assert(err.Error(), gc.Equals, "<path>: conversion to duration: time: invalid duration failure")
+	c.Assert(err.Error(), gc.Equals, "<path>: conversion to duration: time: invalid duration \"failure\"")
 
 	out, err = sch.Coerce(42, aPath)
 	c.Assert(out, gc.IsNil)

